gui/src/components/Panel.svelte

```diff
@@ -78,6 +78,7 @@
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
+    display: inline-block;
```

Comment:
Why?

Reply:
This was necessary to fix the ellipsis overflow for the panel title. Right now it doesn't work for me at all — instead it's just silently truncated.

Reply:
I'm not sure what the underlying problem here was but I think this is the wrong solution - what we want is a `display: block` section that spans all the available space between the left and right action groups.

Could you post a screenshot or something showing the issue you ran into without this change? There may be a better fix.

Reply:
Sure — I leave it up to you to decide the right fix :)
https://user-images.githubusercontent.com/3443024/135058170-d720772e-09e1-415c-b67c-17efc8fff8b5.mp4

Reply:
I see. My first look at the issue makes me think perhaps the solution is `width: calc(100%)`, as strange as that is.

To explain what we _want_ here, the panel header should have its width automatically filling up the width of the panel, and the title portion should take up all the space between both left and right action sections - i.e. `max-content auto max-content` grid column layout.

Apparently `text-overflow: ellipsis` only renders when the width is constrained in absolute units like pixels. However, `calc()` outputs as absolute pixels, so I *think* if we wrap the title in a block with `width: calc(100%)` we can have the `auto` width _and_ the `ellipsis` working properly.

That all being said, the content of this title has been changes so it should usually not overflow anymore as `Workspace af28109rjwefjsdiqq...` has now changed to `my/project/name` where it uses the shortest unique path. Should still overflow sometimes but usually not.